internal/nodes/avi_model_nodes.go

```diff
@@ -1077,6 +1067,38 @@ type AviHostPathPortPoolPG struct {
 	IngName       string
 }
 
+func (v *AviHostPathPortPoolPG) GetCheckSum() uint32 {
+	// Calculate checksum and return
+	v.CalculateCheckSum()
+	return v.Checksum
+}
+
+func (v *AviHostPathPortPoolPG) CalculateCheckSum() {
+	var checksum uint32
+	sort.Strings(v.Path)
+	sort.Strings(v.Host)
```

Comment:
@swathinsankaran: `Host` are marked nil. I think we can skip sorting it.

Reply:
This function is getting called from other places so we cant remove the string sort of hosts.

Reply:
For receiver type `*AviHostPathPortPoolPG`, this will not create a problem as for each object of type `AviHostPathPortPoolPG` , AKO path should be nil as per new code changes.